src/client.h

```diff
@@ -52,18 +52,22 @@ extern std::string      g_host_mac;
 extern int              g_wol_timeout;
 extern bool             g_wol_enabled;
 extern bool             g_KodiLook;
-extern bool             g_eraseIcons;
-extern int16_t          g_timeShiftBufferSeconds;
+extern int g_ServerTimeOffset;
+
+extern int              g_timeShiftBufferSeconds;
```

Comment:
aligned or not aligned?

Reply:
Those were removed in additional commit.  However my philosophy is to only change alignment when I touch the code.